### PR TITLE
Fix l10n process

### PR DESCRIPTION
**Pull request by PVince81**
## Related Issue
Fixes https://github.com/owncloud/phoenix/issues/3012

## How Has This Been Tested?
Run `make l10n-read` and see that the error is gone.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: apps/files/src/store/actions.js [hunk 1]
```diff
@@ -819,12 +819,12 @@ export default {
       .then(_ => {
         context.dispatch('loadFolderSharedWithMe', {
           client: client,
-          $gettext: translate
+          $gettext: $gettext
```
Comment (kulmann) [inline apps/files/src/store/actions.js]: You can just write `$gettext` here, no key: value required.

**Comment by DeepDiver1975** [comment]
It might be good to run make l10n-read in each and every PR to detect if there is something wrong with respect to translations .....

**Comment by PVince81** [comment]
> It might be good to run make l10n-read in each and every PR to detect if there is something wrong with respect to translations .....

The likeliness of this happening is rather low I think, so won't add it.

However, unskipping errors for push/pull would be able to catch a broader scope of potential sync issues in the future.

**Comment by DeepDiver1975** [comment]
> The likeliness of this happening is rather low I think, so won't add it.

Generally speaking: catching an error on PR level is better then days later on the translation sync job (which is monitored by whom? :man_shrugging: )

**Comment by DeepDiver1975** [comment]
and compared to the time we spend on all the acceptance tests this is a rather small job ...

**Comment by DeepDiver1975** [comment]
anyway - your project - your decision :muscle:

**Pull request merged**